src/sot_gepetto_viewer/graph.py

```diff
@@ -54,30 +61,95 @@ def initCmd (self):
 
     def setEntityFilter (self, filter):
         self.entityFilter = filter
-        print self.entityFilter
+
+    def StopRefresh(self):
+        self.timer.stop()
+        print ("Auto Refresh Stopped")
+
+    def LaunchRefresh(self):
+        print ("Auto Refreshed Launched")
+        self.timer = QtCore.QTimer()
+        self.timer.connect(self.timer, QtCore.SIGNAL("timeout()"), self.createAllGraph)
+        self.timer.start(1000)
+    
+    def UpdateFilter (self, filt):
+        self.filter = filt.split()
+
+    def getList (self):
+        import dynamic_graph
+        from dynamic_graph.sot.core.robot_simu import RobotSimu
+
+        chaine = "\n"
+        liste = dynamic_graph.entity.Entity.entityClassNameList
+        for i in liste:
+            chaine += i + "\n"
```

Comment:
FYI, this can be achieved in Python with
```suggestion
        chaine = "\n".join(dynamic_graph.entity.Entity.entityClassNameList)
```

This is not a mandatory change.